sys/solve/test/HelperVexprToSMT.hs

```diff
@@ -51,16 +51,10 @@ createVvar :: VarId -> TXS2SMTVExprTest
 createVvar v =
     TXS2SMTVExprTest (cstrVar v) (toSMTVar v)
 
-createVite :: Set.Set TXS2SMTVExprTest -> TXS2SMTVExprTest -> TXS2SMTVExprTest -> TXS2SMTVExprTest 
-createVite conds (TXS2SMTVExprTest input1 expected1) (TXS2SMTVExprTest input2 expected2) =
-    let listConds = Set.toList (Set.map expected conds) in
-    if length listConds == 1
-    then
-        TXS2SMTVExprTest (cstrIte (Set.toList (Set.map input conds)) input1 input2)
-               ("(ite " ++ head listConds ++ " " ++ expected1 ++ " " ++ expected2 ++ ")")
-    else
-        TXS2SMTVExprTest (cstrIte (Set.toList (Set.map input conds)) input1 input2)
-               ("(ite (and " ++ join " " listConds ++ ") " ++ expected1 ++ " " ++ expected2 ++ ")")
+createVite :: TXS2SMTVExprTest -> TXS2SMTVExprTest -> TXS2SMTVExprTest -> TXS2SMTVExprTest 
```

Comment:
If we adapt existing function maybe it is a good idea to add the missing comments. Otherwise it is difficult for newcomers to understand what is going on.